docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  redis:
+    image: redis
+    container_name: cache
+    expose:
+      - 6379
+  app:
+    build:
+      context: ./
+      dockerfile: ./docker/app/Dockerfile
+    environment:
+      REDIS_HOST: redis
+      HOST: 0.0.0.0
```

Comment:
We might want to update this (and port)  to `APP_HOST` and `APP_PORT` just to make it clear.